cirq-core/cirq/protocols/json_test_data/ConditionalOperation.repr

```diff
@@ -0,0 +1 @@
+cirq.ConditionalOperation(cirq.Y.on(cirq.NamedQubit('target')), [cirq.LineQubit(0), cirq.LineQubit(1)])
```

Comment:
This doesn't seem like a valid repr for this type - `cirq.LineQubit(0)` is not a `MeasurementKey`.